<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>701</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1040</width>
    <height>701</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1040</width>
    <height>701</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1040</width>
    <height>701</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a아시아고딕E</family>
     <pointsize>49</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>전액</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>341</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>490</y>
     <width>321</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a아시아고딕E</family>
     <pointsize>49</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>뒤로가기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>200</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a아시아고딕E</family>
     <pointsize>49</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>2시간</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a아시아고딕E</family>
     <pointsize>49</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>할인(50%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a아시아고딕E</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>아래에서 해당차량을 먼저 눌러주세요</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
